packages/catalog-client/src/types.ts

```diff
@@ -44,6 +44,10 @@ export interface CatalogApi {
     options?: CatalogRequestOptions,
   ): Promise<void>;
 
+  // Attachments
+  getAttachment(name: EntityName, key: string): Promise<Blob>;
```

Comment:
you probably want the return type to be not just Blob, but an object with a `data: Blob` field so that we can fit in more stuff in there later (such as content type, or other related metadata)

Reply:
Not sure for this particular call. Right now this calls the endpoint directly and no metadata is available. Would probably need two calls internally, but I still a good idea.

Reply:
Yeah it's one of those "man I regret not wrapping this" situations that happen a lot of the time down the road :)

Reply:
Might be that response headers are added that we'd be interested in